src/components/BlockFeatured/BlockFeatured.vue

```diff
@@ -122,6 +160,42 @@ export default {
         },
         formattedCredits() {
             return nl2br(this.credits).split("<br>")
+        },
+        formattedCreditsSecondary() {
+            return nl2br(this.creditsSecondary).split("<br>")
+        }
+    },
+    mounted() {
+        if (this.credits) {
+            // Get height of credits block to find how much to translate image
+            this.imageTranslateValue = this.$refs.hover.clientHeight + "px"
+            // Get height of additional credits to set the spacer height on hover
+            this.creditsHeight = this.$refs.secondary.clientHeight + "px"
+        }
+    },
+    methods: {
+        startHover() {
+            if (this.credits) {
+                // Move image up the height of the credits block
+                this.$el.querySelector(
+                    ".media"
+                ).style.transform = `translateY(-${this.imageTranslateValue})`
+                // Animate height of credits spacer to match visible credits
+                this.$el.querySelector(
+                    ".additional-credits"
+                ).style.maxHeight = `${this.creditsHeight}`
+                // Animate background hover state credits up from translated position
+                this.$refs.secondary.style.transform = "none"
```

Comment:
Don't use JS to set the style like this. You want to do this using data and a computed style (kind of like we do for for classes).

So you have the same methods, but the methods sets data() of the height, then you have a computed style, like this: https://github.com/funkhaus/liveinithaca.com/blob/617a38a71d451bc0e99dbb8db9f59cb06f434e39/src/components/block/ExpandedInfo.vue#L45-L51

That way your method just changes a data() val, and everything from there is Vue reactive.

If you do the z-index approach on the credits, then you also won't need the transform: none.

Reply:
@drewbaker Ok, I've switched the ones that have accessible markup in the component, so I can use `:style` in the way you mentioned. But for translating the image, I'm doing that on the `img` element within WpImage, which I don't have access to in the component, and therefore was targeting it using querySelector(). Do you prefer another way of doing this, or does that work?

Reply:
@austinblanchard Why don't you just translate the entire `wp-image`? Why does it need to be `.media` inside it? 

```
        <wp-image
            v-if="image && !isNews"
            class="block-image"
            :image="image"
            mode="fullbleed"
           :style="imageStyles"
        />

       // Computed
       imageStyles() {
            return {
                 transform: translateY(this.imageTranslateValue);
            }
      }
```